docs/use-case2.rst

```diff
@@ -89,7 +106,8 @@ following the format :code:`http://<ip>:<port>`):
 
       curl <url>/v2/entities/String/attrs/data/value -X PUT -s -S --header 'Content-Type: text/plain' --data-binary \"\"
 
-- Execute :code:`integration-service` in the third terminal with the YAML example file:
+- Execute *eProsima Integration-Service* using the :code:`soss` command in the third terminal and with the YAML example
```

Comment:
Same as before, missing link to YAML file. It might happen in the following examples as well

Reply:
ok.

Reply:
Adding TODO.